Ngsa.App.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <TargetFramework>net5.0</TargetFramework>
     <RootNamespace>Ngsa.Application</RootNamespace>
-    <AssemblyName>ngsa-aspnetapp</AssemblyName>
+    <AssemblyName>aspnetapp</AssemblyName>
```

Comment:
Does AssemblyName need to be "aspnetapp" for the customer deployment?  I think I remember that.  If so, leave it, if not let's use ngsa-app since we're using it as the default for the headers if it is run sans a flag.